docs/astro/sql/operators/transform.rst

```diff
@@ -0,0 +1,45 @@
+==================
+transform operator
+==================
+
+.. _transform_operator:
```

Comment:
```suggestion
```

Reply:
Referring to this in the documents will render `When to use the ``transform`` operator` and not `transform operator`.

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/628/commits/647cc41a8a2c1f61fc7307c608e3936c93287652)